app/views/course/virtual_classrooms/_virtual_classroom.html.slim

```diff
@@ -27,9 +27,10 @@
   div.access-link
     - link_exists = virtual_classroom.instructor_classroom_link.present?
     - is_instructor = can?(:edit, virtual_classroom)
+    - is_main_instructor = virtual_classroom.instructor == current_user
```

Comment:
This is probably not very good because "Generate Virtual Classroom Link" button will be needed in both:
1. `if !link_exists? && is_instructor?` block and
2. `elsif link_exists?` block

It's needed in block 2 when current user is not the main instructor